storefront-reference-architecture-master/cartridges/app_storefront_assets_slots/cartridge/models/product/decorators/shipping.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+var ContentMgr = require("dw/content/ContentMgr");
+
+module.exports = function (object, product) {
+    let shipping = "";
+
+    const contentAsset = ContentMgr.getContent(
+        `product-shipping-${product.ID}`
+    );
+
+    if (contentAsset && contentAsset.online) {
+        shipping = contentAsset.custom.body;
+    }
+
+    Object.defineProperty(object, "shipping", {
```

Comment:
Is this the best name for this attribute?

Reply:
fixed